web/src/routes/(user)/sharing/sharedlinks/+page.svelte

```diff
@@ -33,8 +33,8 @@
     const isConfirmed = await dialogController.show({
       id: 'delete-shared-link',
       title: $t('delete_shared_link'),
-      prompt: 'Are you sure you want to delete this shared link?',
-      confirmText: 'Delete',
+      prompt: $t('are_you_sure_you_want_to_delete_this_shared_link'),
```

Comment:
We should probably shorten this very long translations to something like confirm_delete_shared_link
Having them match the current string exactly means if we ever change it we have to change the key too.
Happy to make a commit on top of your PR to change some of these if you like?

Reply:
Fixed